README.md

```diff
@@ -40,8 +40,8 @@ Range, Readings
 
 Establish quality parameters: 
 
-- What is the maximum complexity (CCN) per function? _enter CCN and create corresponding yml in the `.github/workflows` folder
-- How many lines of duplicate code will you tolerate? _enter the number of lines and create corresponding yml in the `.github/workflows` folder
+- What is the maximum complexity (CCN) per function? 3 CCN and create corresponding yml in the `.github/workflows` folder
```

Comment:
Your complexity limit in the yml is 4? Did you decide the complexity first and then code, or did you adjust the complexity according to the code? Which way should it be?

Reply:
Complexity should be decided first!
Will follow it in the upcoming assignment.👍